subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/InstrumentationConfiguration.kt

```diff
@@ -51,7 +52,15 @@ open class InstrumentationConfiguration(val name: String) {
 
     var kubernetesNamespace = "android-emulator"
 
+    /**
+     * It must be a valid reportId.
+     * Get test runs from report by id. Filter already succeed or new tests.
+     * Stay only failed contained in report
+     */
+    var keepFailedTestsFromReport: String? = null
```

Comment:
This property has two responsibility:
- report id - source for tests
- filter condition: only failed tests

What we will do, if we want to filter by other conditions? 
For instance, only lost tests from the same report.

Reply:
Will refactor filters completely in MBS-8045